dscblog/pages/info/login-banner.html

```diff
@@ -0,0 +1,17 @@
+{% if not request.user.is_authenticated %}
+<div style="position:bottom ;
+            width:100%; 
+            height:350px;
+            left: 60px;
+            top:60px;
+            margin-top:20px; 
+            background: radial-gradient(91.06% 87.79% at 7.31% 5.58%, #96FF8B 0%, #A5FFC2 24.18%, #92FFF2 45.35%, #FFCAEF 74.34%, #FFE654 100%);
```

Comment:
make the background colors darker (maybe a different set but darker) for dark mode.
use it like:
```css
[data-theme="dark"] .elem {
  background-color: ...
}
```
Do this inside style tags or css files